edx_proctoring/api.py

```diff
@@ -1488,6 +1488,25 @@ def get_filtered_exam_attempts(course_id, search_by):
     return [ProctoredExamStudentAttemptSerializer(active_exam).data for active_exam in exam_attempts]
 
 
+def get_last_relevant_proctored_practice_attempt(user_id, proctoring_backend, course_id=None):
```

Comment:
The current place this function is used, the course_id is always passed in. Why should we consider course_id is specified case at all in your change? Shouldn't we consider course_id <> the current course_id of the request?